flow/workflows/cdc_flow.go

```diff
@@ -501,21 +501,22 @@ func CDCFlowWorkflow(
 			var panicErr *temporal.PanicError
 			if errors.As(err, &panicErr) {
 				logger.Error(
-					"panic in sync flow",
+					"panic in sync flow, sleeping for 10 minutes",
```

Comment:
does this include nondeterminism?

Reply:
no, non determinism isn't an activity error, but a workflow error which prevents workflow resuming

Reply:
my bad wasn't thinking straight, nondeterminism can only be probably caught by an interceptor